flasher.cpp

```diff
@@ -136,6 +136,8 @@ void Flasher::Init()
     connect(file_downloader_.get(), &file_downloader::FileDownloader::Downloaded, this, &Flasher::FileDownloaded);
     connect(file_downloader_.get(), &file_downloader::FileDownloader::DownloadProgress, this, &Flasher::DownloadProgress);
 
+    connect(&serial_port_, SIGNAL(error(QSerialPort::SerialPortError)), this, SLOT(HandleSerialPortError(QSerialPort::SerialPortError)));
```

Comment:
I've recently bumped into a comment that SLOT/SIGNAL macros are old-style. Would you consider using a new style?
`connect(&serial_port_, &QSerialPort::errorOccurred, this, &Flasher::HandleSerialPortError);`

Reply:
Done, thanks.